src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -113,6 +113,11 @@ class SeekBar extends Slider {
    * @listens mousedown
    */
   handleMouseDown(event) {
+    // only allow left-click seeking
+    if (event.buttons !== 1) {
```

Comment:
shouldn't this be [`0`](https://w3c.github.io/uievents/#dom-mouseevent-button)? `1` is the auxiliary button.

Reply:
for some reason ```button``` not work, then I tried ```buttons``` and it work, then there is funny issue with touchscreen, I'll do a careful test and might delete this pull, it getting messy